spec/controllers/sign_up/select_email_controller_spec.rb

```diff
@@ -64,10 +64,10 @@
     context 'with only one verified email address' do
       let(:user) { create(:user) }
 
-      it 'redirects to the sign up completed path' do
+      it 'loads properly' do
         response
 
-        expect(response).to redirect_to(sign_up_completed_path)
+        expect(response).to be_ok
       end
     end
 
```

Comment:
While I can see some value in regression coverage, I think we can probably just get rid of this if we don't have any special handling for one vs. many